command/app.go

```diff
@@ -69,6 +69,10 @@ var app = &cli.App{
 			Name:  "no-sign-request",
 			Usage: "do not sign requests: credentials will not be loaded if --no-sign-request is provided",
 		},
+		&cli.BoolFlag{
+			Name:  "legacy",
```

Comment:
Can we use a more meaningful flag name instead of `legacy` (such as `--use-v1-api`)? It is hard to understand the context without reading usage text.